apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -45,4 +71,24 @@ export class CommonCartridgeExportService {
 			};
 		});
 	}
+
+	private mapToWebContent(fileInfos: FileDto[]): ICommonCartridgeWebContentProps[] {
+		const webContentProps: ICommonCartridgeWebContentProps[] = [];
+		for (const fileInfo of fileInfos) {
+			webContentProps.push({
+				identifier: this.getIdentifierString(fileInfo.id),
+				href: fileInfo.name,
+				file: Buffer.from(''),
+				parentFolder:
+					fileInfo.parentType === FileRecordParentType.Lesson || fileInfo.parentType === FileRecordParentType.Task
+						? this.getIdentifierString(fileInfo.parentId)
+						: undefined,
+			});
+		}
+		return webContentProps;
+	}
+
+	private getIdentifierString(id: string): string {
```

Comment:
This function can also be used in line 59 and 68. Maybe making this function a static utility function and move it into another class/file?